src/view/hooks/useYfmHtmlBlockStyles.ts

```diff
@@ -0,0 +1,41 @@
+import {useEffect, useState} from 'react';
+
+import {useThemeValue} from '@gravity-ui/uikit';
+
+import {IHTMLIFrameElementConfig} from '../../extensions';
+import {getYfmHtmlBlockCssVariables} from '../hocs/withYfmHtml/utils';
+
+export const useYfmHtmlBlockStyles = () => {
+    const theme = useThemeValue();
+    const [config, setConfig] = useState<IHTMLIFrameElementConfig | undefined>();
+
+    useEffect(() => {
+        const bodyStyles = window.getComputedStyle(document.body);
+
+        const colorTextPrimary = bodyStyles.getPropertyValue('--g-color-text-primary');
+        const colorTextSecondary =
+            theme === 'light' || theme === 'light-hc'
+                ? bodyStyles.getPropertyValue('--yfm-color-hljs-subst')
+                : bodyStyles.getPropertyValue('--g-color-text-primary');
+        const colorBackground = bodyStyles.getPropertyValue('--g-color-base-background');
+        const fontFamily = bodyStyles.getPropertyValue('--g-font-family-sans');
+        const fontSize = bodyStyles.getPropertyValue('--g-text-body-1-font-size');
```

Comment:
we can define mapping for styles/variables
Something like that:
```
colorBackground –> --g-color-base-background
fontFamily –>      --g-font-family-sans
fontSize –>        --g-text-body-1-font-size
```

Reply:
fixed